Section added to Remdesivir (#201)
greenelab/covid19-review: content/20.treatments.md

The effectiveness of Remdesivir for treating patients with COVID-19 is currently under investigation.
Remdesivir has been used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.

> Indeed, the New England Medical Journal published a Gilead Sciences sponsored trial that administered 200mg of Remdesivir intravenously on day 1 followed by a further 100mg/day for 9 days on compassionate-use basis to 61 patients hospitalized with confirmed COVID-19 [@doi:10.1056/NEJMoa2007016].

Reviewer: ```suggestion
A recent publication, led by Gilead Sciences, the maker of Remdesivir, reported outcomes for compassionate-use of the drug administered 200mg of intravenously on day 1 followed by a further 100mg/day for 9 days to 61 patients hospitalized with confirmed COVID-19 [@doi:10.1056/NEJMoa2007016].
```

> Following treatment, data from 53 patients were analyzed and a clinical improvement was observed in 68% (36/53) of patients.

Reviewer: ```suggestion
```

> While that result might seem promising, there were significant issues with the study design that undermine the seemingly positive outcome.

Reviewer: ```suggestion
There were significant issues with the study design or lack thereof.
```

> These included the fact that the trial did not include a randomized control group, some patients only required low doses of oxygen, others required ventilation, patients analyzed had mixed demographics as it was a multisite trial, there was a short follow-up period of investigation, some patients worsened, some patients died, and lastly 8 patients were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.

Reviewer: ```suggestion
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
Patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Some patients worsened, some patients died, and eight were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
```

> As a result of these confounding issues, it is difficult to determine if there was any treatment effect or would these outcomes have occurred regardess of treatment.

Reviewer: ```suggestion
As a result of these issues, though data from 53 patients were analyzed and a clinical improvement was observed in 68% (36/53) of patients, it is difficult to determine if there was any treatment effect or would these outcomes have occurred regardless of treatment.
```

Reviewer 2: ```suggestion
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it cannot be determined whether treatment with Remdesivir had an effect or whether these patients would have recovered regardless of treatment.
```
Rephrasing of @cgreene's suggestion

> Therefore, no conclusive deductions can be made at this time until  randomised controlled trials are completed.

Reviewer: ```suggestion
Therefore, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.
```

Reviewer 2: If you take my suggestion on the previous sentence, here I would suggest using Casey's sentence but swapping "Therefore" with "As a result"

Here, 200mg of Remdesivir was administered intravenously on day 1, followed by a further 100mg/day for 9 days [@doi:10.1056/NEJMoa2007016].
There were significant issues with the study design or lack thereof.
There was no randomized control group.
The inclusion criteria were variable: some patients only required low doses of oxygen, others required ventilation.
The study included many sites, potentially with variable inclusion criteria and treatment protocols.
Patients analyzed had mixed demographics.
There was a short follow-up period of investigation.
Some patients worsened, some patients died, and eight were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
Therefore, even though the study reported clinical improvement in 68% of the 53 patients ultimately evaluated, due to the significant issues with study design, it cannot be determined whether treatment with Remdesivir had an effect or whether these patients would have recovered regardless of treatment.

> Therefore, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.

Author: ```suggestion
As a result, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.
```
